df2gspread/df2gspread.py

```diff
@@ -123,15 +125,15 @@ def upload(df, gfile="/New Spreadsheet", wks_name=None,
         cell_list = wks.range('%s%s:%s%s' % (first_col, start_row, last_col, start_row))
         for idx, cell in enumerate(cell_list):
             cell.value = df.columns.astype(str)[idx]
```

Comment:
Current behavior force conversion to string of all values that goes to gspread, you would need to turn it off.

Reply:
what do you mean?

Reply:
I mean that right now there is this `df.columns.astype(str)` trick

Reply:
Is that the case? That wasn't how I read the docs here: https://developers.google.com/sheets/api/reference/rest/v4/ValueInputOption

where I assumed strings would still be sent, google would then attempt to smart parse the string in the case of `USER_ENTERED`.